frontend/src/store/modules/searchResults.js

```diff
@@ -0,0 +1,66 @@
+import Vue from 'vue'
```

Comment:
You're not using your state anywhere. Is it work in progress?

Reply:
Yes, it is